### PR TITLE
i18n(fr): Update `guides/cms/builderio.mdx` from #9611

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Update `guides/cms/builderio.mdx` from #9611

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/cms/builderio.mdx [hunk 1]
```diff
@@ -283,14 +283,6 @@ export async function getStaticPaths() {
...
     ();
-  return [
-    ...posts.map(({ data: { slug, title } }) => [
-      {
-        params: { slug },
-        props: { title },
-      },
-    ]),
-  ];
 }
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/cms/builderio.mdx:286]:
```suggestion
  return posts.map(({ data: { slug, title } }) => ({
    params: { slug },
    props: { title },
  }))
}
```

**Review by ArmandPhilippot — CHANGES_REQUESTED** [review]
I think you forgot something 😅

File: src/content/docs/fr/guides/cms/builderio.mdx (hunk 1 above)
Comment (thomasbnt) [reply src/content/docs/fr/guides/cms/builderio.mdx:286]: Owwwh the mistake...

**Review by ArmandPhilippot — APPROVED** [review]
LGTM! 😄

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**